terraform/modules/rds-aurora-database/main.tf

```diff
@@ -0,0 +1,99 @@
+// setting the provider
+provider "aws" {
+  version = "~> 2.0"
+  region  = "ap-southeast-2"
+}
+
+
+// defining custom vpc
+// TODO: utilize the project's main VPC
+resource "aws_vpc" "temp_custom_vpc" {
+  cidr_block = "192.168.0.0/16"
+  tags = {
+    "Name" = "Terraform - temp_custom_vpc"
+  }
+}
+
+//defining custom subnet
+// TODO: utilize the project's subnetsh
+resource "aws_subnet" "temp_da_rds_subnet" {
+  vpc_id     = aws_vpc.temp_custom_vpc.id
+  cidr_block = "192.168.51.0/24"
+  availability_zone = "ap-southeast-2a"
+
+  tags = {
+    Name = "Terraform - temp_da_rds_subnet"
+  }
+}
+
+resource "aws_subnet" "temp_da_rds_subnet2" {
+  vpc_id     = aws_vpc.temp_custom_vpc.id
+  cidr_block = "192.168.52.0/24"
+  availability_zone = "ap-southeast-2b"
+
+  tags = {
+    Name = "Terraform - temp_da_rds_subnet2"
+  }
+}
+
+//defining custom subnet for the database
+resource "aws_db_subnet_group" "da_db_subnet_group" {
```

Comment:
I'd recommend using variables.

```
resource "aws_db_subnet_group" "da_db_subnet_group" {
  name             = "${var.name}-db-subnet-group"
  subnet_ids       = var.subnets

  tags = merge(
      var.tags,
      {
        "Name"    = "${var.name}-db-subnet-group"
      }
    )
}
```

Reply:
@zarajoy created the variable `project-name` in her VPC module. I like the variable name and could be used here.
Ideally, when we start to put everything together, we could have multiple environments (prod, test). We would change the project-name to `devops-wordpress-prod` or `devops-wordpress-test` (example names) and it should not have any conflict.